asciidoc-confluence-publisher-client/src/test/java/org/sahli/asciidoc/confluence/publisher/client/ConfluencePublisherTest.java

```diff
@@ -67,6 +67,7 @@
 
```

Comment:
If I see this correctly, there is no test yet covering the `null` version message case both for adding and updating.